plugins/package-managers/swiftpm/src/funTest/kotlin/SwiftPmFunTest.kt

```diff
@@ -88,4 +88,20 @@ class SwiftPmFunTest : WordSpec({
             result.withInvariantIssues().toYaml() should matchExpectedResult(expectedResultFile, definitionFile)
         }
     }
+
+    "Analyzing a definition file without a lockfile without dependencies" should {
```

Comment:
Isn't "without a lockfile" implied by "without dependencies"? If you want to mention both, maybe say "with no dependencies nor lockfile"?